### PR TITLE
docs: installation section

### PR DESCRIPTION
**Pull request by itaysk**
intentionally didn't touch the docker and k8s guides for now.
it's draft because waiting for @rafaeldtinoco to add missing events in kallsyms

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
I created https://github.com/aquasecurity/tracee/pull/3629 so we can remove OS packaging entirely (together with the docs). Will provide a PR  request to your branch soon.

**Comment by rafaeldtinoco** [comment]
Feel free to change https://github.com/itaysk/tracee/pull/3 to anything you prefer. The list is double-checked. LGTM feel free to merge whenever you may like this one as well.

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

File: docs/docs/install/prerequisites.md [hunk 1]
```diff
@@ -1,36 +1,74 @@
...
+Tracee needs non-trivial capabilities to instrument the kernel. The easiest way is run Tracee as "privileged" or "root".  
+
+If you want to run Tracee with "least privileges", here are the required capabilities and justifications:
+
+- Manage eBPF maps limits (`CAP_SYS_RESOURCE`)
+- Load and Attach eBPF programs:
+    - `CAP_BPF`+`CAP_PERFMON` for recent kernels (>=5.8) where the kernel perf paranoid value in `/proc/sys/kernel/perf_event_paranoid` is equal to 2 or less
+    - or `CAP_SYS_ADMIN` otherwise
+- `CAP_SYS_PTRACE` (to collect information about processes)
+- `CAP_NET_ADMIN` (to use tc for packets capture)
```
Comment (yanivagman) [inline docs/docs/install/prerequisites.md:64]: We don't use tc anymore. However, other bpf network programs require it as well

**Pull request merged**